install/kubernetes/cilium/values.yaml.tmpl

```
@@ -3333,6 +3333,16 @@ clustermesh:
   # maximum allocatable cluster-local identities.
   # Supported values are 255 and 511.
   maxConnectedClusters: 255
+  remoteCluster:
+    # -- Enable revocation of cached state for a remote cluster once its TTL expires.
+    # When disabled (default), revocation is not performed, but log messages are
+    # emitted when the cache would have been revoked.
+    enableCacheRevocation: false
+    # -- The time to live for the cache of a remote cluster after connectivity is
+    # lost. If the connection is not re-established within this duration, the
+    # cached data is revoked to prevent stale state. If not specified or set to
+    # 0s, the cache is never revoked.
+    cacheTTL: "15m"
```

The fact that it's in a `remoteCluster` dict looks slightly off to me, if we were to have something like this for instance the `maxtConnectedClusters` should probably be in there too?

maybe something like that would be slighly better (?):
```suggestion
  cacheRevocation:
    # -- Enable revocation of cached state for a remote cluster once its TTL expires.
    # When disabled (default), revocation is not performed, but log messages are
    # emitted when the cache would have been revoked.
    enabled: false
    # -- The time to live for the cache of a remote cluster after connectivity is
    # lost. If the connection is not re-established within this duration, the
    # cached data is revoked to prevent stale state. If not specified or set to
    # 0s, the cache is never revoked.
    ttl: "15m"
```